frontend/package.json

```diff
@@ -14,7 +14,8 @@
     "react-canvas-confetti": "1.2.1",
     "sass": "^1.72.0",
     "styled-components": "^6.1.8",
-    "usehooks-ts": "^2.14.0"
+    "usehooks-ts": "^2.14.0",
+    "zustand": "^4.5.2"
```

Comment:
need to ensure this is included in the root package.json too, then we can merge 👍